src/react-native/17-offline-support/offline-support.md

```diff
@@ -48,7 +48,7 @@ This Hook returns an object containing details about the network status.
 ✏️ Run:
 
 ```bash
-npm install @react-native-community/netinfo@11
+npm install --save-dev @react-native-community/netinfo@11
```

Comment:
I think this should be a regular dependency because it’s used in the app (and not just for local dev / testing).